### PR TITLE
add get start kubeflow with icp

### LINKED ISSUE #333: Added document for how to run kuebflow on IBM Cloud Private
[IBM Cloud Private](https://www.ibm.com/cloud/private) is an Enterprise Kubernetes distribution, we will try to add a getting started guide for this.

### PR DESCRIPTION
**Pull request by jinchihe**
fixes #333 

The PR is to add document for how to run kuebflow on IBM Cloud Private.

/cc @gyliu513

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/684)

### CODE REVIEW

File: content/docs/started/getting-started-icp.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
++++
+title = "ICP For Kubeflow"
```
Comment (gyliu513) [inline content/docs/started/getting-started-icp.md]: Do not mention ICP, we should always use the long name "IBM Cloud Private"

File: content/docs/started/getting-started-icp.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+Following [installation steps in IBM Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.0/installing/install.html) to install IBM Cloud Private master, proxy, worker, and optional management and Vulnerability Advisory nodes in your cluster in standard or high availability configurations.
+
+After installing, you can check the ICP after installation:
+
+```
+# kubectl get node
+NAME         STATUS    ROLES                      AGE       VERSION
+10.43.0.38   Ready     management                 11d       v1.11.1+icp-ee
+10.43.0.40   Ready     va                         11d       v1.11.1+icp-ee
+10.43.0.44   Ready     etcd,master,proxy,worker   11d       v1.11.1+icp-ee
```
Comment (gyliu513) [inline content/docs/started/getting-started-icp.md]:
```
10.43.0.44   Ready     etcd,master,proxy,worker   11d       v1.11.1+icp-ee
10.43.0.46   Ready     etcd,master,proxy,worker   11d       v1.11.1+icp-ee
10.43.0.49   Ready     etcd,master,proxy,worker   11d       v1.11.1+icp-ee
```

Please do not set master node as worker node.

File: content/docs/started/getting-started-icp.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+   ```
+   export K8S_NAMESPACE=kubeflow
+   kubectl create namespace $K8S_NAMESPACE
+   ```
+   - *K8S_NAMESPACE* is namespace name that the kubeflow will be installed in. By default should be "kubeflow".
+
+2. Create image policy for the namespace.
+
+   The image policy defination file (image-policy.yaml) is as following:
```
Comment (gyliu513) [inline content/docs/started/getting-started-icp.md]: s/image-policy.yaml/`image-policy.yaml`

File: content/docs/started/getting-started-icp.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+         policy: null
+   ```
+   Create ImagePolicy for the specified namespace.
+   ```
+   kubectl create -n $K8S_NAMESPACE -f image-policy.yaml 
+   ```
+3. Create PV for kubeflow components.
+   
+   Some kubeflow components need PVs to storage data, such as minio, mysql, katib. We need to create PVs for those pods in advance. 
+   The persistent volume defination file (pv.yaml) is as following:
```
Comment (gyliu513) [inline content/docs/started/getting-started-icp.md]: s/pv.yaml/`pv.yaml`

File: content/docs/started/getting-started-icp.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
++++
+title = "ICP For Kubeflow"
+description = "Get Kubeflow running on IBM Cloud Private (ICP)"
+weight = 2
++++
+
+This guide is a quick start to deploy Kubeflow on [IBM Cloud Private (ICP)](https://www.ibm.com/cloud/private).  ICP is an enterprise PaaS layer for developing and managing on-premises, containerized applications. It is an integrated environment for managing containers that includes the container orchestrator Kubernetes, a private image registry, a management console, and monitoring frameworks.
```
Comment (gyliu513) [inline content/docs/started/getting-started-icp.md]: Please also mention the IBM Cloud Private version here as well.

File: content/docs/started/getting-started-icp.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+    Create PV by running below command:
+    ```
+    kubectl create -f pv.yaml
+    ```
+
+### Installing Kubeflow
+
+Follow these steps to deploy Kubeflow:
+
+1. Download a `kfctl` release from the [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/).
```
Comment (gyliu513) [inline content/docs/started/getting-started-icp.md]: Please also mention the version for kfctl as well.

File: content/docs/started/getting-started-icp.md [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+   export K8S_NAMESPACE=kubeflow
+   kubectl create namespace $K8S_NAMESPACE
+   ```
+   - *K8S_NAMESPACE* is namespace name that the kubeflow will be installed in. By default should be "kubeflow".
+
+2. Create image policy for the namespace.
+
+   The image policy defination file (image-policy.yaml) is as following:
+
+    ```
```
Comment (gyliu513) [inline content/docs/started/getting-started-icp.md]: use `yaml` format

File: content/docs/started/getting-started-icp.md [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+10.43.0.40   Ready     va                         11d       v1.11.1+icp-ee
+10.43.0.44   Ready     etcd,master,proxy,worker   11d       v1.11.1+icp-ee
+10.43.0.46   Ready     etcd,master,proxy,worker   11d       v1.11.1+icp-ee
+10.43.0.49   Ready     etcd,master,proxy,worker   11d       v1.11.1+icp-ee
+```
+### Creating image policy and persistent volume
+
+1. Create Kubernetes namespace.
+
+   ```
```
Comment (gyliu513) [inline content/docs/started/getting-started-icp.md]: use `shell` format

File: content/docs/started/getting-started-icp.md [hunk 9]
```diff
@@ -0,0 +1,193 @@
...
+   Create ImagePolicy for the specified namespace.
+   ```
+   kubectl create -n $K8S_NAMESPACE -f image-policy.yaml 
+   ```
+3. Create PV for kubeflow components.
+   
+   Some kubeflow components need PVs to storage data, such as minio, mysql, katib. We need to create PVs for those pods in advance. 
+   The persistent volume defination file (pv.yaml) is as following:
+   
+    ```
```
Comment (gyliu513) [inline content/docs/started/getting-started-icp.md]: use `yaml` format

File: content/docs/started/getting-started-icp.md (hunk 1 above)
Comment (jinchihe) [reply content/docs/started/getting-started-icp.md]: Updated.

File: content/docs/started/getting-started-icp.md [hunk 10]
```diff
@@ -0,0 +1,194 @@
...
+title = "IBM Cloud Private for Kubeflow"
+description = "Get Kubeflow running on IBM Cloud Private"
+weight = 2
++++
+
+This guide is a quick start to deploy Kubeflow on [IBM Cloud Private](https://www.ibm.com/cloud/private) 3.1.0 or later.  IBM Cloud Private is an enterprise PaaS layer for developing and managing on-premises, containerized applications. It is an integrated environment for managing containers that includes the container orchestrator Kubernetes, a private image registry, a management console, and monitoring frameworks.
+
+### Prerequisites
+
+   - Get the system requirements from [IBM Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.0/supported_system_config/hardware_reqs.html) for IBM Cloud Private.
```
Comment (gyliu513) [inline content/docs/started/getting-started-icp.md]: I think this can start at the beginning?

**Comment by jinchihe** [comment]
@gyliu513  Your great comments have been addressed. Thanks a lot!

File: content/docs/started/getting-started-icp.md [hunk 11]
```diff
@@ -0,0 +1,194 @@
...
+
+After installing, you can check the IBM Cloud Private after installation:
+
+```bash
+# kubectl get node
+NAME         STATUS    ROLES                      AGE       VERSION
+10.43.0.38   Ready     management                 11d       v1.11.1+icp-ee
+10.43.0.39   Ready     master                     11d       v1.11.1+icp-ee
+10.43.0.40   Ready     va                         11d       v1.11.1+icp-ee
+10.43.0.44   Ready     etcd,proxy,worker   11d       v1.11.1+icp-ee
```
Comment (gyliu513) [inline content/docs/started/getting-started-icp.md]: Please put `etcd, proxy` to `master` and remove them for `worker` node.

File: content/docs/started/getting-started-icp.md [hunk 12]
```diff
@@ -0,0 +1,194 @@
...
+
+### Prerequisites
+
+   - Get the system requirements from [IBM Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.0/supported_system_config/hardware_reqs.html) for IBM Cloud Private.
+  
+   - Setup NFS Server and export one or more path for persistent volume.
+
+### Installing IBM Cloud Private
+
+Following [installation steps in IBM Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.0/installing/install.html) to install IBM Cloud Private 3.1.0 or later with master, proxy, worker, and optional management and vulnerability advisory nodes in your cluster in standard or high availability configurations.
```
Comment (gyliu513) [inline content/docs/started/getting-started-icp.md]: Highlight that you are using `IBM Cloud Private 3.1.0` in this tutorial.

**Comment by jinchihe** [comment]
@sarahmaddox Your comments have been addressed. Thank you!  
Could you please help to re-generate the `Preview` since updated some format. That's strange, that's OK for markdown in my VC code. :-)

**Comment by jinchihe** [comment]
@sarahmaddox Thank you! Your comments has been addressed.
Could you please help to re-generate the `Preview` since updated some format errors. That's strange, it's OK for markdown in my editor. :-)

**Comment by jinchihe** [comment]
@sarahmaddox Thank you! Your comments has been addressed.
Could you please help to re-generate the `Preview` since updated some format errors. That's strange, it's OK for markdown in my editor. :-)

**Comment by sarahmaddox** [comment]
Hallo @jinchihe, Netlify automatically creates and refreshes the preview for you every time you create a PR and push updates to it. To find the preview:

* Wait for the automated PR workflow to do some checks. When it's ready, you should see a comment like this: **deploy/netlify — Deploy preview ready!**
* Click **Details** to the right of "Deploy preview ready" to see a preview of your updates. 

Let me know when you're ready for a final review.

**Comment by sarahmaddox** [comment]
@jinchihe following on from my earlier comment about how to find the preview: You can also build the docs locally, and see the updates rendered on your local site as you type. See the guide in the docs README: https://github.com/kubeflow/website

**Comment by jinchihe** [comment]
@sarahmaddox Thanks a lot! That's OK now in the `preview`. Could you please review and approve if no any more comments?

**Comment by sarahmaddox** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**